app/assets/javascripts/application.js

```diff
@@ -7,7 +7,6 @@
 //= require pikaday
 //= require chartjs
 //= require select2
-//= require charts
```

Comment:
chart-js-rails(0.0.6) needs to require `//= require Chart`.
Please see [here](https://github.com/coderbydesign/chart-js-rails/tree/v0.0.6).